### PR TITLE
Introduce a new nuget package, Google.Protobuf.Tools, basically to contain protoc on multiple platforms.

### PR DESCRIPTION
**Pull request by jskeet**
I've moved both protoc.exe and the proto files out of Google.Protobuf.
The .proto files aren't a slam-dunk, but it feels like they belong with protoc as you'd *use* them with protoc.
It's not clear to me whether we really need both an x86 and x64 version of protoc.exe, as x86 would work on 64-bit Windows anyway. Discuss :)

### CODE REVIEW

File: csharp/Google.Protobuf.Tools.nuspec [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <file src="..\src\google\protobuf\any.proto" target="tools\google\protobuf" />
+    <file src="..\src\google\protobuf\api.proto" target="tools\google\protobuf" />
+    <file src="..\src\google\protobuf\descriptor.proto" target="tools\google\protobuf" />
+    <file src="..\src\google\protobuf\duration.proto" target="tools\google\protobuf" />
+    <file src="..\src\google\protobuf\empty.proto" target="tools\google\protobuf" />
+    <file src="..\src\google\protobuf\field_mask.proto" target="tools\google\protobuf" />
+    <file src="..\src\google\protobuf\source_context.proto" target="tools\google\protobuf" />
+    <file src="..\src\google\protobuf\struct.proto" target="tools\google\protobuf" />
+    <file src="..\src\google\protobuf\timestamp.proto" target="tools\google\protobuf" />
+    <file src="..\src\google\protobuf\any.proto" target="tools\google\protobuf" />
```
Comment (dalebrubaker) [inline csharp/Google.Protobuf.Tools.nuspec]: a nit - line 34 duplicate of line 25?

Comment (jskeet) [inline csharp/Google.Protobuf.Tools.nuspec]: Thanks! Fixed - looks like it was a bug in the previous nuspec.

**Comment by jtattermusch** [comment]
Thanks!

**Comment by jtattermusch** [comment]
Also, if you wish to publish this package for Google.Protobuf.Tools for   3.0.0-beta2 release retroactively,  you can basically grab all protoc flavors from https://www.nuget.org/packages/Grpc.Tools/0.13.1,
as for that release gRPC is pinned to protobuf 3.0.0-beta2  (https://github.com/grpc/grpc/tree/release-0_13_1/third_party) so versions will match.

**Comment by jtattermusch** [comment]
Also, will we want to eventually drop protoc.exe from Google.Protobuf nuget?

**Pull request merged**